tests/Microsoft.Identity.Test.SideBySide/UnifiedCacheTests.cs

```diff
@@ -35,6 +35,8 @@
 using msal::Microsoft.Identity.Client;
 using Microsoft.Identity.Test.LabInfrastructure;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using msal::Microsoft.Identity.Client.AppConfig;
```

Comment:
>msal [](start = 6, length = 4)

interesting. Was this supposed to move to a M.I.C namespace instead of it's own namespace?

Reply:
There's a separate issue for moving out AppConfig/ApiConfig namespaces which will be in a separate PR.